bot/exts/moderation/alts.py

```diff
@@ -0,0 +1,166 @@
+import discord
+from discord.ext import commands
+from pydis_core.site_api import ResponseCodeError
+from pydis_core.utils.members import get_or_fetch_member
+
+from bot import constants
+from bot.bot import Bot
+from bot.converters import UnambiguousMemberOrUser
+from bot.log import get_logger
+from bot.pagination import LinePaginator
+from bot.utils.channel import is_mod_channel
+from bot.utils.time import discord_timestamp
+
+log = get_logger(__name__)
+
+
+class AlternateAccounts(commands.Cog):
+    """A cog used to track a user's alternative accounts across Discord."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def error_text_from_error(error: ResponseCodeError) -> str:
+        """Format the error into a user-facing message."""
+        if resp_json := error.response_json:
+            errors = ", ".join(
+                resp_json.get("non_field_errors", []) +
+                resp_json.get("source", []) +
+                resp_json.get("detail", [])
+            )
+            if errors:
+                return "Error(s) from site: " + errors
+            return str(error.response_json)
+        return error.response_text
+
+    async def alts_to_string(self, alts: list[dict[str: str | int | list[int]]]) -> list[str]:
+        """Convert a list of alts to a list of string representations."""
+        lines = []
+        guild = self.bot.get_guild(self.bot.guild_id)
+        for idx, alt in enumerate(alts):
+            alt_obj = await get_or_fetch_member(guild, alt["target"])
+            alt_name = str(alt_obj) if alt_obj else alt["target"]
+            created_at = discord_timestamp(alt["created_at"])
+            updated_at = discord_timestamp(alt["updated_at"])
+
+            edited = f" edited on {updated_at}\n" if "edited" in alt else "\n"
+            lines.append(
+                f"**Association #{idx} - {alt_name}**\n"
+                f"<@{alt['target']}> - {alt['target']}\n"
+                f"Issued by: <@{alt['actor']}> ({alt['actor']}) on {created_at}{edited}"
+                f"Context: {alt['context']}\n"
+                f"<@{alt['target']}> has {len(alt['alts'])} associated account(s)"
```

Comment:
How about `gettext.ngettext` to pluralize this like a king?

Reply:
Is [this](https://github.com/python-discord/bot/compare/1152d3c5e57db137e6420634ee4e8c6b28cffee8..56d8fe9fec46b8b31462fac927ee9a966208784c) what you were expecting?

Reply:
That looks like pluralization done right. A great move by the Python Discord Community